sdk/nodejs/ec2/vpcIpv4CidrBlockAssociation.ts

```diff
@@ -0,0 +1,89 @@
+// *** WARNING: this file was generated by the Pulumi Terraform Bridge (tfgen) Tool. ***
```

Comment:
Huh - I guess I missed this file in the PR that generated these new types?

Reply:
Seems like it :)